User/Library/UserPcd.c

```diff
@@ -50,4 +50,4 @@ BOOLEAN  _gPcd_FeatureFlag_PcdFatReadOnlyMode                        = _PCD_VALU
 UINT32   _gPcd_BinaryPatch_PcdSerialRegisterStride                   = 0;
 UINT8    _gPcd_FixedAtBuild_PcdUefiImageFormatSupportNonFv           = 0x00;
 UINT8    _gPcd_FixedAtBuild_PcdUefiImageFormatSupportFv              = 0x03;
-UINT32   _gPcd_FixedAtBuild_PcdImageProtectionPolicy                 = 0x00;
```

Comment:
This PCD is also used by BaseTools and such a modification will break [EDK2_REDUCE_FW_SIZE](https://github.com/acidanthera/audk/blob/master/BaseTools/Conf/tools_def.template#L50-L54) builds, because BaseTools will reject PE files with SectionAlignment < 4KBi.

Reply:
I'm confused, these are the versions of the Pcds used when building userspace tools, which I'm taking in general (unless there's a reason not too, for fuzzing or something else) should match the values used in the main OC build. I'm not clear how these affect or relate to EDK-II/UDK BaseTools?

Reply:
(Btw, my specific reason for _wanting_ to change this, was to make AppleEfiSignTool be less surprising, by matching what it does and doesn't accept to what Apple signature checking in live OC accepts.)

Reply:
These PCDs are used by [ImageTool](https://github.com/acidanthera/audk/blob/master/BaseTools/ImageTool/GNUmakefile#L26). Maybe it is time to eradicate audk dependency on OpenCore.

Reply:
Oh! Yes, there seem to be two quite different purposes here, with just one set of Pcds. Thank you.

Reply:
- We should assign desired values in ImageTool and possibly other tools in OpenCore.
- Then we should update default values here.

For the latter we should decide on the defaults.

* I think supporting only PE and not UE by default is fair.
* Also enabling image protection policy is a good idea, but I vote for all sources (i.e. not just fw, option roms but also user supplied images).